index.html

```diff
@@ -9,6 +9,19 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="container">
+    <div class="top-item">
+      <h1>Título del libro</h1><p>(género)</p>
```

Comment:
what do you think ?? 
```suggestion
      <h1>Título del libro<span>(género)</span></h1>
```

Reply:
the card wording is about Page Titles, Paragraph, Headings, Margin or padding. I would think that he did it well. 🤔